handbook/ce/customer-support-onboarding.md

```diff
@@ -1,5 +1,204 @@
+* Add Sourcegraph as [a browser search engine](https://docs.sourcegraph.com/integration/browser_search_engine). To search our private code, log in to our internal dogfood instance (`k8s.sgdev.org`) and add another entry: `https://k8s.sgdev.org/search?q=%s`.
+* Install a text editor of your choice.
+ 
+* Start monitoring Zendesk for new support issues and try to own at least one by the end of the week. It is okay if we miss SLA for these first few cases. Take your time trying to figure out the right initial response, following our [workflow](https://about.sourcegraph.com/handbook/ce/support-workflow) steps. This will feel painful and tedious for a bit -- this is a reflection of our team being so nascent and the opportunity we have to make this better (not of your ability). Work as a team to solve issues, run responses by each other, etc. The more activity we have in our #customer-support-chat Slack channel, the better. And, if you have not used Zendesk before and would like a crash course, just let the team know in our #customer-support-chat Slack channel and we will make it happen!
+
+Big pause here. These next tasks need a ton of improvement -- as mentioned in the last bullet! They are all supposed to help you learn the product and get more comfortable doing the job. Work with what is here and take abundant notes for how this can be improved.
+
+* Browse some of the available materials to help you learn more about the product:
+	* [Campaigns Overview Deck](https://docs.google.com/presentation/d/1CN3KQf1Hfdb4RO6FgBgKuiHK4ERcOAHPgVnOcBu-MPU/edit#slide=id.g7d2aea8729_0_0)
+	* [Structural Search Overview](https://zoom.us/rec/share/CJtwQ7uEp3v1pvPqdUD7GDuaYm_2g6w3zSP7GNA3aGQHZDjQ72awYXvHEnwsoio6.Bt-0DuuAZjs7UXMc?startTime=1606237440000)
+	* Code Intelligence [video](https://drive.google.com/file/d/1TyCj62LLmhvamXK-CC6D8-7uGk1jmsKj/view) and [slides](https://docs.google.com/presentation/d/181oMTXRmcTqTCfOe5P__fLKwlzV1uk9wvwG6ocFgz20/edit?usp=sharing)
+	* [“Deployment types - A crash course in Sourcegraph”](https://docs.google.com/presentation/d/1u4mbXjubQqV-6WFbuS7Q1b_X6BVh-_GWzzFQMcrAzLw/edit#slide=id.p)
+	* [Architecture overview](https://docs.sourcegraph.com/dev/background-information/architecture)
+	* [https://kube.academy/](https://kube.academy/)
+* Read about [search queries](https://docs.sourcegraph.com/code-search) and perform your first searches.
+* Work through the questions from the [Sales Onboarding Quiz](https://about.sourcegraph.com/handbook/sales/onboarding/quiz) to make sure you understand key concepts. Feel free to skip any obvious answers and discuss any questions you have in our #customer-support-chat Slack channel.
```

Comment:
Could also be a relative link if you want it to be (../sales/onboarding/quiz.md)